### DISCUSSION TITLE
Does FastDDS have java bindings for Java/Android applications to use fastdds?

### DISCUSSION THREAD

**Discussion by chunyisong** [eProsima/Fast-DDS#3187] [Q&A]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

FastDDS can generate java bindings code.

### Current behavior

I do not konw How FastDDS generate java bindings code.

### Steps to reproduce

FastDDS can generate java bindings code.

### Fast DDS version/commit

all

### Platform/Architecture

Ubuntu Focal 20.04 amd64, Windows 10 Visual Studio 2019, MacOS Mojave 10.14

### Transport layer

Default configuration, UDPv4 & SHM, TCPv4, TCPv6, Shared Memory Transport (SHM), Intra-process, Data-sharing delivery, Zero copy

**Comment by JLBuenoLopez** [comment]
Hi @chunyisong 

The proper forum to ask questions about Fast DDS is not this one. According to [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#qa) the Q&A discussion forum is where this issue should be. I would proceed and move it.

Also, I take the opportunity to answer your question: no,  there are no java bindings for Fast DDS as of now. You might contact [eProsima commercial support](https://github.com/eProsima/Fast-DDS#getting-help) if you are interested on introducing this into Fast DDS roadmap.
